source/includes/webhooks/_candidate_events.md

```diff
@@ -1202,3 +1202,66 @@ See web hook [common attributes](#common-attributes).
+| `credited_to` | For each of this candidate's application, destroy `application.credited_to`. |
+| `custom_application_fields` | For each of this candidate's applications, destroy `application.custom_fields`. |
+| `custom_candidate_fields` | Destroy all values from `candidate.custom_fields` |
+| `custom_rejection_question_fields` | For each of this candidate's application, destroy all custom fields on rejection details. |
+| `current_company` | Set `candidate.company` to null. |
+| `current_title` | Set `candidate.title` to null. |
+| `education` | Remove all values from `candidate.educations` |
+| `email_addresses` | Destroy all values from `candidate.email_addresses` |
+| `employment` | Remove all values from `candidate.employments` |
+| `emails` | Destroy record of any e-mails sent to this candidate via Greenhouse. |
+| `follow_up_reminders` | Destroy any follow-up reminders configured on this person. |
+| `full_name` | Set `candidate.first_name` to "Anonymized" and `candidate.last_name` to the `candidate.id`. |
+| `headline` | Set the candidate's headline to null. |
+| `inmails` | Destroy record of any LinkedIn Inmails sent to this candidate via Greenhouse. |
+| `innotes` | Destroy record of any LinkedIn Innotes sent to this candidate via Greenhouse. |
+| `location` | For each of this candidate's applications, remove the `location` information. |
+| `notes` | Destroy all notes on this candidate. |
+| `offers` | Destroy all information on all offers on any of this candidate's applications without destroying that an offer was made. |
+| `phone_numbers` | Remove all values from `candidate.phone_numbers` |
+| `prospect_jobs` | Remove all associations between this prospect and jobs. |
+| `prospect_offices` | Remove all associations between this prospect and offices. |
+| `prospect_offices_and_departments` | Remove all associations between this prospect and departments. |
+| `prospect_owner` | For each of this candidate's applications, destroy the prospect owner. |
+| `prospect_pool_and_stage` | For each of this candidate's application, destroy the prospect pool and the prospect pool stage. |
+| `recruiter` | Set `candidate.recruiter` to null |
+| `referral_questions` | Destroy all referral answers for this candidate. |
+| `rejection_notes` | Destroy all notes specifically related to this candidate's rejection. |
+| `rejection_reasons` | Set `rejection_reason` on all candidate applications to null. |
+| `scorecards_and_interviews` | Destroy all scorecards with this candidate id and all interviews with any application from this candidate id |
+| `social_media_links` | Remove all values from `candidate.social_media_addresses` |
+| `source` | Remove `application.source` in all this candidate's applications. |
+| `tags` | Remove all values from `candidate.tags` |
+| `touchpoints` | Destroy all touchpoints on the candidate, all the candidate's applications, and any notes associated with touchpoints. |
```

Comment:
I'm unclear what "all the candidate's applications" means in this context. It seems to make sense like this unless I'm missing something.

"Destroy all touchpoints on the candidate and any notes associated with touchpoints."

Reply:
Touchpoints can be on a candidate or an application. I wasn't exactly sure the best way to describe this since they're not available in harvest or web hooks anyway.